tests/CMakeLists.txt

```
@@ -287,3 +287,32 @@ ENDIF()
 # Set up the default attest configuration file:
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests/attest.conf.in
   ${CMAKE_BINARY_DIR}/attest.conf)
+
+# For the time being, run `attest` as if it were a single test.
+set(disabled_tests)
+list(APPEND disabled_tests "mpirun=[5-9]")
```

```suggestion
list(APPEND disabled_tests "mpirun=[3-9]")
```
There are probably some other MPI settings we can tweak but for now we need to disable tests past 2 cores.

---

Done